libs/cardano-ledger-core/testlib/Test/Cardano/Ledger/Core/KeyPair.hs

```diff
@@ -110,15 +115,53 @@ deriving instance Typeable r => Eq (KeyPair r)
 instance ToExpr (KeyPair r) where
   toExpr (KeyPair x y) = Tree.App "KeyPair" [toExpr x, Tree.App (take 10 (show y)) []]
 
-mkAddr :: (KeyPair 'Payment, KeyPair 'Staking) -> Addr
-mkAddr (payKey, stakeKey) = Addr Testnet (mkCred payKey) (StakeRefBase $ mkCred stakeKey)
+class MakeCredential c r where
+  mkCred :: c -> Credential r
```

Comment:
```suggestion
  mkCredential :: c -> Credential r
```
How about getting rid of the abbreviation while we're at it

Reply:
Sure, why not. I am in favor of better naming

Reply:
Done